src/Atom/AtomParser.cs

```diff
@@ -93,7 +93,7 @@ public ISyndicationContent ParseContent(string value)
 
         public virtual bool TryParseValue<T>(string value, out T result)
         {
-            return Converter.TryParseValue<T>(value, out result);
+            return AtomConverter.TryParseValue<T>(value, out result);
```

Comment:
AtomConverter.TryParseValue<T> doesn't seem to be necessary.

Reply:
I believe AtomConverter.TryParseValue would be used for reading an Atom feed.  As I mentioned above, AtomConverter has replaced Converter.

Converter.TryParseValue tries to parse dates in many different formats, and it's debatable whether that's the correct approach or whether to just support the format required in the Atom spec, and the format required in the RSS spec, etc. (as in your comment above.)

Reply:
A greater compatibility is "relaxed on input and strict on output". It was intentional to parse extended date-time formats with Atom and Rss formatters, because there are feeds not following the spec sharply. Out output should follow strictly the spec. 
Many thanks to @chriswill for correctly pointing the issue with RFC 3339 compliance, his PR and help addressing the issue.